### ISSUE TITLE
Usage example might be misleading

### ISSUE THREAD

**Issue by Itaros** [LukasBoersma/FastCGI#5]
I believe basic usage example(and corresponding method docs as well) are a bit incorrect. It currently states that the way to return 200 OK is like this:
```CSharp
var responseString =
          "HTTP/1.1 200 OK\n"
        + "Content-Type:text/html\n"
        + "\n"
        + "Hello World!"; 
```
Shouldn't it be more in line of?:
```CSharp
var responseString =
          "Status: 200 OK\n"
        + "Content-Type:text/html\n"
        + "\n"
        + "Hello World!"; 
```
As far as I am aware proxy should process Status as header and process its provided status code accordingly(delegating to requesting client as formal PROTOCOL CODE TEXT\n sequence by default). Currently it is hardcoding http\1.1 as output and breaks delegation of status code other than 200 at least with nginx.

**Comment by sebgod** [comment]
@Itaros, yes I think this is correct, as (Fast)CGI is not the one sending back HTTP protocol strings, but the underlying web server does.

**Comment by LukasBoersma** [comment]
Thanks for reporting this. It sounds like you are correct. It has been a while that I last looked at this library, so I will check with the spec and fix this when I have time.
